viewer/packages/camera-manager/src/ComboControls.ts

```diff
@@ -131,17 +142,17 @@ export default class ComboControls extends EventDispatcher {
 
   public update = (deltaTime: number): boolean => {
     const {
-      camera,
-      target,
-      targetEnd,
-      spherical,
-      sphericalEnd,
-      deltaTarget,
+      _camera,
+      _target,
+      _targetEnd,
+      _spherical,
+      _sphericalEnd,
+      _deltaTarget,
       handleKeyboard,
       enableDamping,
       dampingFactor,
       EPSILON,
-      targetFPS,
+      _targetFPS,
       enabled
     } = this;
```

Comment:
This isn't important, but for the next time I'd change this into 
```
const {
      _camera: camera,
      _target: target,
      _targetEnd: targetEnd,
      _spherical: spherical,
...
    } = this;
```
to avoid all the changes below.

Reply:
Didn't know about that one, will use in the future